.pre-commit-config.yaml

```diff
@@ -41,7 +41,7 @@ repos:
     rev: v1.1.0
     hooks:
       - id: detect-secrets
-        args: [ '--baseline', '.secrets.baseline', '--exclude-files', 'tests/*/cassettes/*' ]
+        args: [ '--baseline', '.secrets.baseline', '--exclude-files', 'tests/*/cassettes/*', --exclude-files, 'jupyterlab/dashboards/*' ]
```

Comment:
the `--exclude-file` is duplicated and you forgot the "`"

Reply:
`$ detect-secrets scan --exclude-files '.*\.signature$' --exclude-files '.*/i18n/.*'`
In the readme example it says to use it twice. I could not figure out how to make it work only using it once. https://github.com/Yelp/detect-secrets#filters.